mod-infra/src/main/scala/at/forsyte/apalache/infra/passes/options.scala

```diff
@@ -1,47 +1,99 @@
-package at.forsyte.apalache.infra.passes
+package at.forsyte.apalache.infra.passes.options
 
-/** Collects the passoptions supported for configuring Apalache's various passes */
-object options {
```

Comment:
Realized we can declare the package just by putting it in the file and giving the package name, so no need for the object.